<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElevationSettings</class>
 <widget class="QWidget" name="ElevationSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>elevation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="heightBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <double>-1000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="heightLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>height [m]</string>
        </property>
        <property name="buddy">
         <cstring>heightBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="sSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="slopeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>slope
percentage</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="slopeSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
